bitnami/nginx-ingress-controller/templates/controller-hpa.yaml

```
@@ -1,6 +1,6 @@
 {{- if and .Values.autoscaling.enabled (eq .Values.kind "Deployment") }}
 {{- if semverCompare "<1.23-0" (include "common.capabilities.kubeVersion" .) }}
-apiVersion: autoscaling/v2beta1
+apiVersion: {{ include "common.capabilities.hpa.apiVersion" . }}
 {{- else }}
 apiVersion: autoscaling/v2
 {{- end }}
```

If/else are no longer needed:
```suggestion
apiVersion: {{ include "common.capabilities.hpa.apiVersion" . }}
```